src/domain/Inputs/SingleDateInput/MonthPicker.tsx

```diff
@@ -0,0 +1,63 @@
+import moment, { Moment } from 'moment'
+import React from 'react'
+
+import { Variables } from '../../../common'
+import { GridLayout } from '../../Layouts/GridLayout'
+import { HorizontalAlignment } from '../../Layouts/GridLayout/style'
+
+const monthPickerYears = () => {
+  const years = []
+  for (let i = moment().year() - 100; i <= moment().year() + 100; i++) {
+    years.push(<option value={i}>{i}</option>)
+  }
+  return years
+}
+
+const MonthPicker = ({date, handleDateChange}: {date?: Moment | null, handleDateChange: (date: Moment | null) => void}) => {
+  const value = date ?? moment()
+  const selectYear = (e: any) => {
```

Comment:
You should be able to find a good type for that select element event right?

Reply:
oh yeah that was some wip code that i forgot about

Reply:
ill fix that